Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.lean

```diff
@@ -85,11 +85,18 @@ def FinStronglyMeasurable [Zero β]
   ∃ fs : ℕ → α →ₛ β, (∀ n, μ (support (fs n)) < ∞) ∧ ∀ x, Tendsto (fun n => fs n x) atTop (𝓝 (f x))
 
 /-- A function is `AEStronglyMeasurable` with respect to a measure `μ` if it is almost everywhere
-equal to the limit of a sequence of simple functions. -/
+equal to the limit of a sequence of simple functions.
+
+One can specify the sigma-algebra according to which simple functions are taken using the
+`AEStronglyMeasurable[m]` notation in the `MeasureTheory` scope. -/
 @[fun_prop]
-def AEStronglyMeasurable
-    {_ : MeasurableSpace α} (f : α → β) (μ : Measure α := by volume_tac) : Prop :=
-  ∃ g, StronglyMeasurable g ∧ f =ᵐ[μ] g
+def AEStronglyMeasurable [m : MeasurableSpace α] {m₀ : MeasurableSpace α} (f : α → β)
+    (μ : Measure[m₀] α) : Prop :=
```

Comment:
Is it possible to restore the `volume_tac` default argument? (and remove the `volume` measure you added in a few places)
```suggestion
def AEStronglyMeasurable [m : MeasurableSpace α] {m₀ : MeasurableSpace α} (f : α → β)
    (μ : Measure[m₀] α := by volume_tac) : Prop :=
```

Reply:
Oh yes of course. Not sure how I removed it